src/components/FormField/FormField.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable react/require-default-props */
+/* eslint-disable react/no-unstable-nested-components */
+import React from 'react';
+import QuestionCircle from '../../icons/regular/QuestionCircle';
+import Tooltip from '../Tooltip/Tooltip';
+import Label from '../Label/Label';
+
+interface RequiredProp {
+  required: boolean;
+  label: string;
+}
+interface FormFieldProps {
+  label?: string;
+  required?: RequiredProp;
```

Comment:
`required` sounds more like a boolean in an input. Maybe is a more descriptive way to call this props.